File: 6-kyu/highest-scoring-word/javascript/solution.js
function high(x){
  const values = x.split(' ').map(word => {
    let total = 0;
    for (let i = 0; i < word.length; i++)
      total += word[i].charCodeAt() - 96;
    return total;
    });
  return x.split(' ')[values.indexOf(Math.max(...values))];  
}

File: 6-kyu/highest-scoring-word/ruby/solution.rb
def high(x)
  words = x.split
  word_values = words.map(&:chars).map { |arr| arr.map { |c| c.ord - 96 }.reduce(&:+) }
  max_idx = word_values.index(word_values.max)
  words[max_idx]
end